sharder/sharded-bot.js

```diff
@@ -100,6 +103,23 @@ const manager = new ShardingManager(startup, {
   token,
 });
 
-manager.on('shardCreate', (shard) => console.log(`Shard ${shard.id} launched`));
+manager.on('shardCreate', (shard) => {
+  shard.on('reconnecting', () => {
+    console.log(`Reconnecting shard: [${shard.id}]`);
+  });
+  shard.on('spawn', () => {
+    console.log(`Spawned shard: [${shard.id}]`);
+  });
+  shard.on('ready', () => {
+    console.log(` Shard [${shard.id}] is ready`);
+  });
+  shard.on('death', () => {
+    console.log(`Died shard: [${shard.id}]`);
+  });
+  shard.on('error', (err) => {
+    console.log(`Error in  [${shard.id}] with : ${err} `);
+    shard.respawn();
+  });
```

Comment:
This seems like better wording for the log plus no need for the shard.respawn() since sharding manager already has an option for it. See https://discord.js.org/#/docs/discord.js/main/typedef/ShardingManagerOptions which also already has a default value of true.

```suggestion
  shard.on('death', () => {
    console.log(`Shard [${shard.id}] down`);
  });
  shard.on('error', (err) => {
    console.log(`Error in Shard [${shard.id}]: ${err} `);
  });
```